### PR TITLE
Enable linting *.postcss and *.pcss files

### LINKED ISSUE #101: Postcss files lint
Please check *.pcss and *.postcss too.

### PR DESCRIPTION
**Pull request by juwain**
Fixes #101.

### CODE REVIEW

**Comment by Arcanemagus** [comment]
Can you add a spec testing a PostCSS file?

**Comment by Arcanemagus** [comment]
If you need help just let us know :wink:.

**Comment by juwain** [comment]
Yep, I need help with test specs, because I didn't developed for Atom before. Is there an instruction or tutorial for writing test specs?

**Comment by Arcanemagus** [comment]
At it's core the specs are run through [Jasmine 1.3](http://jasmine.github.io/1.3/introduction.html) with a tiny bit of customization (primarily `waitsForPromise` being added).

At the simplest what I would recommend (and what I would do if implementing this) is:

* Add a PostCSS folder + file to the `fixtures` folder.
* Place the path to that file as a variable at the top of the specs file, such as [this](https://github.com/AtomLinter/linter-stylelint/blob/v2.3.0/spec/linter-stylelint-spec.js#L9)
* Then just copy a block like [this one](https://github.com/AtomLinter/linter-stylelint/blob/v2.3.0/spec/linter-stylelint-spec.js#L31-L43), modifying it slightly to account for the PostCSS file's contents.

**Comment by juwain** [comment]
OK, thanks, will make it soon.

**Comment by Arcanemagus** [comment]
Looks like AppVeyor barfed, going to try closing/reopening this to clear it up :confused:.

**Comment by Arcanemagus** [comment]
Can you rebase this instead of merging?

**Comment by juwain** [comment]
Sorry, I'll try.

**Comment by Arcanemagus** [comment]
If you need help just let me know and I can try to walk you through it :wink:.

**Comment by juwain** [comment]
OK, I'm a bit confused of this merge, can you help me to resolve situation? 😊

File: spec/linter-stylelint-spec.js [hunk 1]
```diff
@@ -42,15 +47,15 @@ describe('The stylelint provider for Linter', () => {
     );
   });
 
-  it('bundles and works with stylelint-config-standard', () => {
```
Comment (onedesert) [inline spec/linter-stylelint-spec.js]: Here is are missing base stylelint-config-standard. Should be stylelint-config-standard and less stylelint-config-standard assertions.

**Comment by onedesert** [comment]
@juwain I've PR to your feature with fix merge https://github.com/juwain/linter-stylelint/pull/1

**Comment by juwain** [comment]
OK, thank you very much, @onedesert

**Comment by juwain** [comment]
OK, sorry for this mess, I should learn rebasing upstreams, sure. 😔

**Comment by Arcanemagus** [comment]
Was thinking about this and as all we care about is that the scope is correct having an identical file with just a different extension doesn't really help test anything. All that matters here is that the relevant `language-____` package is putting the right scope on the file. So you can just remove one of those files (sorry!) as it's just doing the same work over again for something that is beyond the control of this plugin anyway.

**Comment by Arcanemagus** [comment]
And don't worry @juwain, you have to start learning somewhere, and GitHub now lets you do squash merges quite easily so all these commits will just get squashed together in the end :wink:.

**Comment by Arcanemagus** [comment]
This LGTM btw, other than the duplicated work as stated above.

**Comment by juwain** [comment]
Thanks, @Arcanemagus!

**Comment by Arcanemagus** [comment]
CI is green, and everything looks good. Merging it in :grinning:.

**Comment by Arcanemagus** [comment]
Published in v2.4.0 :tada:!

**Pull request merged**